packages/fx-core/src/core/index.ts

```diff
@@ -265,52 +266,58 @@ export class FxCore implements v3.ICore {
       const projectSettings: ProjectSettings = {
         appName: appName,
         projectId: inputs.projectId ? inputs.projectId : uuid.v4(),
-        solutionSettings: {
-          name: "",
-          version: "1.0.0",
-        },
         version: getProjectSettingsVersion(),
         isFromSample: false,
       };
-      ctx.projectSettings = projectSettings;
-      const createEnvResult = await this.createEnvWithName(
-        environmentManager.getDefaultEnvName(),
-        projectSettings,
-        inputs
-      );
-      if (createEnvResult.isErr()) {
-        return err(createEnvResult.error);
-      }
+      if (isCreatedFromExistingApp(inputs)) {
+        //TODO create from existing Tab or Bot/ME
+        // 1. call App Studio V3 API to create manifest with placeholder
+        // 2. create config.local.json to store existing App information
+      } else {
+        // there is no solution settings if created from existing app
```

Comment:
Looks like this comment is for the `if` case, which is created from existing app.

Reply:
Yes, will be moved later